src/test/Drips.t.sol

```diff
@@ -106,6 +106,82 @@ contract DripsTest is DSTest {
         return recv(recv(recv1, recv2, recv3), recv4);
     }
 
+    function random(bytes32 name, uint256 salt) internal pure returns (uint256) {
+        return random(name, salt, type(uint256).max);
+    }
+
+    function random(
+        bytes32 name,
+        uint256 salt,
+        uint256 range
+    ) internal pure returns (uint256) {
+        return uint256(keccak256(abi.encode(name, salt))) % range;
+    }
+
+    function combineSalt(uint256 a, uint256 b) public pure returns (uint256) {
+        return uint256(keccak256(abi.encode(a, b)));
+    }
+
+    function genRandomRecv(
+        uint256 randomSalt,
+        uint8 amountReceiver,
+        uint128 maxAmtPerSec,
+        uint32 maxStart,
+        uint32 maxDuration
+    ) internal pure returns (DripsReceiver[] memory) {
+        uint256 inPercent = 100;
+        uint256 probDefaultEnd = random("default.end", randomSalt, inPercent);
+        uint256 probStartNow = random("start.now", randomSalt, inPercent);
+        DripsReceiver[] memory receivers;
+        for (uint8 i = 0; i < amountReceiver; i++) {
+            uint256 salt = combineSalt(randomSalt, i);
+            uint256 amtPerSec = random("amtPerSec", salt, maxAmtPerSec) + 1;
+            uint256 start = random("start", salt, maxStart);
+            if (start % 100 <= probStartNow) start = 0;
+            uint256 duration = random("duration", salt, maxDuration);
+            if (duration % 100 <= probDefaultEnd) duration = 0;
+
+            receivers = recv(receivers, recv(uint256(i), amtPerSec, start, duration));
```

Comment:
Yes, I mean it is clear that it is slower but I thought it would be insignificant for the tests.  However, you are right with a lot of receivers and a high number of fuzz runs. 

I guess I just wanted to use your nice helper functions.  :smiley_cat: 

I will change it.